build.gradle

```diff
@@ -1,10 +1,41 @@
-subprojects {
-  buildscript {
-    repositories {
-      jcenter()
-      google()
-    }
+buildscript {
+  ext.versions = [
+        'compileSdk': 27,
+        'minSdk': 14,
+        'targetSdk': 27,
+        'sourceCompatibility': JavaVersion.VERSION_1_7,
+        'targetCompatibility': JavaVersion.VERSION_1_7,
+        'okhttp': '3.10.0',
+        'supportLibrary': '27.1.0'
+    ]
+
+  ext.deps = [
+        androidPlugin      : 'com.android.tools.build:gradle:3.0.1',
+        okhttp             : "com.squareup.okhttp3:okhttp:${versions.okhttp}",
+        mockWebServer      : "com.squareup.okhttp3:mockwebserver:${versions.okhttp}",
+        pollexor           : 'com.squareup:pollexor:2.0.4',
+        supportV4          : "com.android.support:support-v4:${versions.supportLibrary}",
```

Comment:
Also support-v4 isn't a thing anymore. We should depend on a granular library. What's this used for?

Reply:
android.support.v4.app.NotificationCompat
android.support.v4.app.FragmentActivity
android.support.v4.widget.CursorAdapter
android.support.v4.app.LoaderManager
android.support.v4.content.CursorLoader
android.support.v4.content.Loader
android.support.v4.app.Fragment

Reply:
so compat and fragment? that's not too bad.

Reply:
I removed the dependency from the `picasso` module.